dataproc/create_cluster.py

```diff
@@ -14,22 +14,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This sample walks a user through creating a Cloud Dataproc cluster using
+# the Python client library.
+
+# [START dataproc_create_cluster]
```

Comment:
I think the TODO(developer) uncomment and set idiom is more applicable to when the region tag is inside the function and therefore the function signature is unset. If the region tag is outside of the function, then the user will see the variable both in the function signature and in the comments. It's not a big deal but if you can think of another way to communicate that the user is responsible for filling those in (perhaps by moving the comments outside of the function definition to the top level, or something) that might help.